components/automate-gateway/api/deployment/deployment.proto

```diff
@@ -49,3 +57,15 @@ message ServiceVersion {
 	string version = 3;
 	string release = 4;
 }
+
+message DeploymentStatusRequest {}
+
+message DeploymentStatusResponse {
```

Comment:
There is a legacy `StatusResponse` and I wasn't sure whether I could repurpose `StatusResponse`, hence `DeploymentStatusResponse`.

Reply:
Mind pointing me to where the legacy version is?

Reply:
@stevendanna It's https://github.com/chef/automate/blob/master/components/automate-gateway/api/legacy/legacy.proto